examples/graphql-cache-invalidation/graphql_cache_invalidation.md

```diff
@@ -0,0 +1,23 @@
+# GraphQL service - Cache invalidation
+
+The Ballerina `graphql` module provides functionality for cache invalidation. The `invalidate()` and `invalidateAll()` APIs in the `graphql:Context` can be used to invalidate caches in a `graphql:Service`. To invalidate a specific cache in a `resource` function, the `invalidate()` API requires to specify the name of the resource. On the other hand, the `invalidateAll()` API invalidates all caches within the service.
```

Comment:
We need to explain how to provide the path when we have a field deeper in the graph. @DimuthuMadushan Can you check this?

Use `resource method` instead of `resource function`.